packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -129,6 +130,15 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final Color? collapsedBackgroundColor;
 
+  /// The blend mode applied to the [backgroundColor] or [collapsedBackgroundColor].
+  ///
+  /// If no [backgroundBlendMode] is provided then the default painting blend
+  /// mode is used.
```

Comment:
What is the default? And what setting will result in the expected behavior from the issue?

Reply:
I was having troubles testing these changes in something like dartpad to see which value would solve my issue specifically. Can I DM you on discord and you can help me with that?